### PR TITLE
download process added

### PR DESCRIPTION
**Pull request by mablue**
## Summary
I need this cuz of that when it take more time to download I don't know how much time will I wait and how much pair is downloaded

## What's new?
A process text like that: (50/100) after pair name when downloading data

### CODE REVIEW

**Comment by Killakill90** [comment]
[]()

**Comment by MelnikovAG** [comment]
Way

**Comment by mablue** [comment]
> Way

cuz I wait just 4hours for download all data and I dont know how much will I wait and howmuch of it done!

**Comment by MelnikovAG** [comment]
Way

**Comment by MelnikovAG** [comment]
Get a way

**Comment by mablue** [comment]
> Get a way

you go there its the way :)

9pq95b42d7k51

It is Russia way

File: freqtrade/data/history/history_utils.py [hunk 1]
```diff
@@ -117,10 +117,11 @@ def refresh_data(datadir: Path,
     :param timerange: Limit data to be loaded to this timerange
     """
     data_handler = get_datahandler(datadir, data_format)
-    for pair in pairs:
+    for idx, pair in enumerate(pairs):
+        process = f' ({idx}/{len(pairs)})'
         _download_pair_history(pair=pair, timeframe=timeframe,
                                datadir=datadir, timerange=timerange,
-                               exchange=exchange, data_handler=data_handler)
+                               exchange=exchange, data_handler=data_handler, process=process)
```
Comment (MelnikovAG) [inline freqtrade/data/history/history_utils.py]: up

**Review by MelnikovAG — APPROVED** [review]

File: freqtrade/data/history/history_utils.py (hunk 1 above)
Comment (mablue) [reply freqtrade/data/history/history_utils.py]: https://github.com/freqtrade/freqtrade/pull/5432/commits/d2484496dfe2e70fcea84ee0371a510efcc374f4

**Comment by mablue** [comment]
:|

**Comment by xmatthias** [comment]
Please use the [developer setup](https://www.freqtrade.io/en/latest/developer/#developer-setup) to learn how to run the tests (pytest, flake8 and mypy) locally to avoid unnecessary "try to fix this" commits.

**Comment by mablue** [comment]
thanks for help @xmatthias 
I got this:
```
______________________________________________________________________ test_talib_bollingerbands_near_zero_values ______________________________________________________________________

    def test_talib_bollingerbands_near_zero_values():
        inputs = pd.DataFrame([
            {'close': 0.00000010},
            {'close': 0.00000011},
            {'close': 0.00000012},
            {'close': 0.00000013},
            {'close': 0.00000014}
        ])
        bollinger = ta.BBANDS(inputs, matype=0, timeperiod=2)
>       assert bollinger['upperband'][3] != bollinger['middleband'][3]
E       assert 1.25e-07 != 1.25e-07

tests/test_talib.py:14: AssertionError
=================================================================================== warnings summary ===================================================================================
```
and this:
```
__________________________________________________________________________ test_load_data_with_new_pair_1min ___________________________________________________________________________

ohlcv_history_list = [[1511686200000, 8.794e-05, 8.948e-05, 8.794e-05, 8.88e-05, 0.0877869], [1511686500000, 8.88e-05, 8.942e-05, 8.88e-05, 8.893e-05, 0.05874751], [1511686800000, 8.891e-05, 8.893e-05, 8.875e-05, 8.877e-05, 0.7039405]]
mocker = <pytest_mock.plugin.MockerFixture object at 0x7f3cf8308040>, caplog = <_pytest.logging.LogCaptureFixture object at 0x7f3cf8322a30>
default_conf = {'ask_strategy': {'order_book_top': 1, 'use_order_book': False}, 'bid_strategy': {'ask_last_balance': 0.0, 'check_dept...book': False}, 'cancel_open_orders_on_exit': False, 'datadir': '/home/mab/Documents/DEV/freqtrade/tests/testdata', ...}
tmpdir = local('/tmp/pytest-of-mab/pytest-4/test_load_data_with_new_pair_10')

    def test_load_data_with_new_pair_1min(ohlcv_history_list, mocker, caplog,
                                          default_conf, tmpdir) -> None:
        """
        Test load_pair_history() with 1 min timeframe
        """
        tmpdir1 = Path(tmpdir)
        mocker.patch('freqtrade.exchange.Exchange.get_historic_ohlcv', return_value=ohlcv_history_list)
        exchange = get_patched_exchange(mocker, default_conf)
        file = tmpdir1 / 'MEME_BTC-1m.json'
    
        # do not download a new pair if refresh_pairs isn't set
        load_pair_history(datadir=tmpdir1, timeframe='1m', pair='MEME/BTC')
        assert not file.is_file()
        assert log_has(
            'No history data for pair: "MEME/BTC", timeframe: 1m. '
            'Use `freqtrade download-data` to download the data', caplog
        )
    
        # download a new pair if refresh_pairs is set
        refresh_data(datadir=tmpdir1, timeframe='1m', pairs=['MEME/BTC'],
                     exchange=exchange)
        load_pair_history(datadir=tmpdir1, timeframe='1m', pair='MEME/BTC')
        assert file.is_file()
>       assert log_has_re(
            'Download history data for pair: "MEME/BTC" (1/1), timeframe: 1m '
            'and store in .*', caplog
        )
E       assert False
E        +  where False = log_has_re('Download history data for pair: "MEME/BTC" (1/1), timeframe: 1m and store in .*', <_pytest.logging.LogCaptureFixture object at 0x7f3cf8322a30>)

tests/data/test_history.py:135: AssertionError
---------------------------------------------------------------------------------- Captured log call -----------------------------------------------------------------------------------
```

how fix them? binance have not meme/btc and why my ta-lib didint calculate true values!!?I use Arch linux and installed it with 
```
yay -S ta-lib
```

**Comment by xmatthias** [comment]
`test_talib_bollingerbands_near_zero_values` is not related to your change - but stems from an installation error (ta-lib was not installed with the BTC fix). (when in doubt, try to run the tests against develop).

test_load_data_with_new_pair_1min - apparently the log-message changed / was reworded. If that's intentional, you'll simply have to  adapt the test ...

**Comment by mablue** [comment]
> `test_talib_bollingerbands_near_zero_values` is not related to your change - but stems from an installation error (ta-lib was not installed with the BTC fix). (when in doubt, try to run the tests against develop).

thanks for help I think it make me bunch of problems in strategy development too! wow this test is amazing.I think cuz of that I compile this in my pc with default config.I hope Im in true assumption...

> test_load_data_with_new_pair_1min - apparently the log-message changed / was reworded. If that's intentional, you'll simply have to adapt the test ...

yes I edited this cuz I want to add an (1/1) to log.but there is not a Meme/btc coin in my binance account and when I make a new one bunch of other problems happen!(I eddited the name of one data with new name Meme/btc but it also make errors. also I changed meme/btc in test code to xrp/btc and it make errors too :))
Its very hard to undrestand but yes it will easy to remove some lines :))

**Comment by AchmadFathoni** [comment]
> `test_talib_bollingerbands_near_zero_values` is not related to your change - but stems from an installation error (ta-lib was not installed with the BTC fix). (when in doubt, try to run the tests against develop).
> 
> test_load_data_with_new_pair_1min - apparently the log-message changed / was reworded. If that's intentional, you'll simply have to adapt the test ...

I got same testing error when using conda environment at 78e129034. My current ta-lib version:
```
libta-lib                 0.4.0                h516909a_0    conda-forge
ta-lib                    0.4.24                   pypi_0    pypi
```
How do I install ta-lib with BTC fix?

**Comment by AchmadFathoni** [comment]
Well I think I got it, I will try to make a PR about installation document for ta-lib in conda.

**Pull request closed without merging**